app/src/main/java/com/company/strengthtracker/domain/util/NormalizeLists.kt

```diff
@@ -0,0 +1,15 @@
+package com.company.strengthtracker.domain.util
+
+class NormalizeLists {
+    fun normalize(
+        max: Float,
+        min: Float,
+        list: MutableList<Float>
+    ): MutableList<Float> {
+        var l: MutableList<Float> = mutableListOf<Float>()
+        list.forEachIndexed { i, it ->
+            l.add((it - min) / (max - min))
+        }
+        return l
```

Comment:
Consider trying this in the future

```{
return mutableListOf<Float>().apply {
     forEachIndexed { i, it -> ......}
}
}```

Reply:
also if the index isn't being used should be safe to switch this to just `forEach`